setup.ts

```diff
@@ -106,6 +106,49 @@ async function mongoDB(): Promise<void> {
   }
 }
 
+//image check for size and format
+
+async function imageValidation(): Promise<void> {
+  const { maxImageSize, maxImageWidth, maxImageHeight, supportedImageFormats } =
+    await inquirer.prompt([
+      {
+        type: "input",
+        name: "maxImageSize",
+        message: "Enter the maximum allowed image size in bytes:",
+        default: process.env.MAX_IMAGE_SIZE || "5242880",
+      },
+      {
+        type: "input",
+        name: "maxImageWidth",
```

Comment:
1. If a value is already in the `.env` file, then it must be seen on the screen, like the other functions in the script.
2. The script must not fail if any of these expected configuration values are not present